ml-agents/mlagents/trainers/trainer.py

```diff
@@ -208,6 +196,8 @@ def write_summary(self, global_step, delta_train_start, lesson_num=0):
                 if self.is_training and self.get_step <= self.get_max_steps
                 else "Not Training."
             )
+            # step = min(self.get_step, self.get_max_steps)
```

Comment:
Do you still need to take the min here?

Reply:
This was part of a set of changes around how we use steps that I think can / should be reverted.  Will fix.